RankensteinApp/App.js

```diff
@@ -79,36 +93,40 @@ class NewsfeedScreen extends React.Component {
       page: 1,
       seed: 1,
       error: null,
-      refreshing: false
+      refreshing: false,
     };
   }
-  static navigationOptions = {
-    header: null
-	
-  }
 
+  
   componentDidMount() {
-    this.makeRemoteRequest();
-  }
+        // do stuff while splash screen is shown
+        // After having done stuff (such as async tasks) hide the splash screen
+        this.makeRemoteRequest();
 
+        if (Platform.OS === 'android') {
+            this.backButtonListener = BackHandler.addEventListener('hardwareBackPress', () => {
+                BackHandler.exitApp()
```

Comment:
You should be consistent with your semicolons. Either use them all the time or never